dev/TabView/TabView.h

```diff
@@ -18,7 +18,10 @@ class TabViewTabCloseRequestedEventArgs :
     public ReferenceTracker<TabViewTabCloseRequestedEventArgs, winrt::implementation::TabViewTabCloseRequestedEventArgsT, winrt::composing, winrt::composable>
 {
 public:
-    TabViewTabCloseRequestedEventArgs(winrt::IInspectable const& item, winrt::TabViewItem tab) { m_item = item; m_tab.set(tab); }
+    TabViewTabCloseRequestedEventArgs(winrt::IInspectable const& item, winrt::TabViewItem tab) : m_item(item)
+    {
+        m_tab.set(tab);
```

Comment:
The event args are short-lived and don't risk creating a cycle so they don't need to be ReferenceTracker or tracker_ref. Can you downgrade them? Just do:
```c++
TabViewTabCloseRequestedEventArgs : public winrt::implementation::TabViewTabCloseRequestedEventArgsT<TabViewTabCloseRequestedEventArgs>
```

That should simplify these classes a bit.

Reply:
Sure, can do.